### ISSUE TITLE
Error importing XLSX, as it's considered as DBF file

### ISSUE THREAD

**Issue by programmdesign** [jazzband/tablib#237]
When I try to import the attached XLSX file with ...

```
import tablib
my_dataset = tablib.Dataset().load(open('simpletest.xlsx').read())
```

... I get the following error, which suggests that tablib consideres the file to be a DBF file.

```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/christoph/.virtualenvs/storesunited/lib/python2.7/site-packages/tablib/core.py", line 443, in load
    format = detect_format(in_stream)
  File "/Users/christoph/.virtualenvs/storesunited/lib/python2.7/site-packages/tablib/core.py", line 1142, in detect_format
    if fmt.detect(stream):
  File "/Users/christoph/.virtualenvs/storesunited/lib/python2.7/site-packages/tablib/formats/_dbf.py", line 83, in detect
    _dbf = dbf.Dbf(StringIO(stream), readOnly=True)
  File "/Users/christoph/.virtualenvs/storesunited/lib/python2.7/site-packages/tablib/packages/dbfpy/dbf.py", line 135, in __init__
    self.header = self.HeaderClass.fromStream(self.stream)
  File "/Users/christoph/.virtualenvs/storesunited/lib/python2.7/site-packages/tablib/packages/dbfpy/header.py", line 135, in fromStream
    _fld = fields.lookupFor(_data[11]).fromString(_data, _pos)
  File "/Users/christoph/.virtualenvs/storesunited/lib/python2.7/site-packages/tablib/packages/dbfpy/fields.py", line 455, in lookupFor
    return _fieldsRegistry[typeCode]
KeyError: 's'
```

I'm working on Mac OS 10.11.2. with openpyxl 2.3.3 and tablib 0.11.3 and Python 2.7.10.

When I open the file as follows, the import works just fine.

```
import tablib
my_dataset = tablib.Dataset()
my_dataset.xlsx = open('simpletest.xlsx').read()
my_dataset[0]
# (u'Test',)
```

[simpletest.xlsx](https://github.com/kennethreitz/tablib/files/201428/simpletest.xlsx)

Is there a fix, or am I missing something?

**Comment by Xuefeng-Zhu** [comment]
A quick fix to this issue is pass 'xlsx' as second argument to specify the format.

**Comment by Xuefeng-Zhu** [comment]
This bug is caused by https://github.com/kennethreitz/tablib/issues/230

**Comment by claudep** [comment]
Should be fixed by ca8dbcf9bea46607d2
